jsk_pcl_ros_utils/scripts/install_sample_data.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+from jsk_data import download_data
+
+
+def main():
+    PKG = 'jsk_pcl_ros_utils'
+
+    download_data(
+        pkg_name=PKG,
+        path='sample/data/2015-11-04-19-37-29_baxter-kiva-object-in-hand-cloud.tgz',
```

Comment:
We should seperate pointcloud_to_pcd.test to sample_pointcloud_to_pcd.launch and pointcloud_to_pcd.test? OK?